specs/experimental/op-contracts-manager.md

```diff
@@ -233,9 +234,11 @@ This method reverts if the upgrade is not successful for any of the chains.
 The high level logic of the upgrade method is as follows:
 
 1. The Upgrade Controller Safe will `DELEGATECALL` to the `OPCM.upgrade()` method.
-2. For each `_systemConfig`, the list of addresses in the chain is retrieved.
-3. For each address:
-   1. If it is receiving new state variables (only the SystemConfig for Isthmus), a call is made to:
+2. The SuperchainConfig contract will be upgraded, if not yet done.
+3. The ProtocolVersions contract will be upgraded, if not yet done.
+4. For each `_systemConfig`, the list of addresses in the chain is retrieved.
+5. For each address:
+   1. If it is receiving new state variables (only the AnchorStateRegistry for Isthmus), a call is made to:
```

Comment:
I think we can make this more generic and avoid focusing on a particular upgrade.

```suggestion
   1. If it is receiving new state variables, a call is made to:
```